### ISSUE TITLE
Vehicle Avitar

### ISSUE THREAD

**Issue by tesladan** [Kemmey/Kemmey-TeslaWatch-Public#97]
I can’t find any documentation on how to specify vehicle avatars. I have a blue model 3 with aero wheels and a red model S 100 D myths on a carbon 18 inch wheels.

**Comment by GeeRee1** [comment]
To set the image of your Tesla as a background, you must enter a URL. Enter for this the following URL in the setting: https://static-assets.tesla.com/v1/compositor/. Activate the Add vehicle option codes querystring toggle, and your Tesla with the correct configuration will be used as background image.

**Comment by GeeRee1** [comment]
That is from https://tm3een.home.blog/2020/06/27/apple-watch-app-for-tesla/

enter the URL using the keyboard icon on the watch which coordinates with the phone, much easier to type.

**Comment by Kemmey** [comment]
And in next version, custom urls Will be phased out entirely - just flip the background image toggle on and that's it :-D

**Comment by tesladan** [comment]
Good to hear.
Followed the instructions below with no success. (No background image) Is this because I have 2 cars? (MS & M3)
Screenshots from settings...


To set the image of your Tesla as a background, you must enter a URL. Enter for this the following URL in the setting: https://static-assets.tesla.com/v1/compositor/. Activate the Add vehicle option codes querystring toggle, and your Tesla with the correct configuration will be used as background image.

 👤 Daniel Lane
📧   dan@daniellane.net
📱 (360) 606-LANE mobile
          (360) 606-(5263)

🌀 Sent from my iPhone Xs⚡️

**Comment by GeeRee1** [comment]
Both these need to be toggled on ....
It worked for my multiple cars....


![91BB9047-F2ED-49B9-8361-90BB378E9849](https://user-images.githubusercontent.com/69560014/90706401-0e6ebf00-e263-11ea-9a16-dc0b114014d8.png)

**Comment by Kemmey** [comment]
Yeah - the settings are shared across all vehicles. But if you set the url to the compositor url posted above and turn on querystring, each vehicle should get its respective background image when you switch. 

If not, please post image of sample background url which can be seen in the settings as well - or wait for the next release. All I need for release is the last few translations and approval from Apple :-D

**Comment by Vchoovers** [comment]
Loving the app so far!  Thank you.  

Minor issue. 

Phone Settings: background image is activated. On the home screen I see my Blue Model 3.   I assumed that would show on the watch. 
Watch: Background Image is active, but I see nothing. I put the Icon on the middle section hoping the car avatar would appear and nothing. It just says Watch App For Tesla. 
I could find where the URL should pop up.  Or, the Add Vehicle options slider shown above. 

I have toggled off and on Background Image on both watch and phone to no avail. :(

Thanks, in advance for your help!!

**Comment by Kemmey** [comment]
hey @Vchoovers 

The background image feature is now baked into the app and is just a toggle - no more configuring the urls :-D

The background image should show up both on your phone and on your watch - what do you see if you tap your vehicle name on the main screen? If background image is enabled, you should definitely see your vehicle image there, as well as on the background of the main screen and controls screens.

**Comment by 2018Quicksilver** [comment]
New to the app.  Watch series 3 with LTE, WatchOS 8.7.1.  iOS 16.4.  My car image shows up perfectly but the car name is a generic “Model 3”.   How do I get the app to display my car name?  Have already rebooted the watch and still shows Model 3 for car name.

**Comment by 2018Quicksilver** [comment]
![IMG_1100](https://user-images.githubusercontent.com/130124834/230542269-4a254fc2-d07e-47d1-9f50-7112206eeb62.jpeg)
